CHANGES

```diff
@@ -6,6 +6,7 @@ Dependencies
 
 * Drop python 3.5 support
 * Drop docutils 0.12 and 0.13 support
+* LaTeX: add ``tex-gyre`` font dependency
```

Comment:
I guess this should be added to sphinx-doc/docker-ci?

Reply:
Yes, please.

Reply:
Done PR at https://github.com/sphinx-doc/docker-ci/pull/5